data/35e/wizards_of_the_coast/supplement/complete_arcane/car_spells.lst

```diff
@@ -1,4 +1,4 @@
-# CVS $Revision: $ $Author: $ -- Sun Aug 17 15:41:15 2014 -- reformated by prettylst.pl v1.51 (build 24906)
+# CVS $Revision: $ $Author: $ -- Sun Aug 17 15:41:15 2014 -- reformatted by prettylst.pl v1.51 (build 24906)
```

Comment:
Inconsequential fix, since the same issue will crop up unless the source of the issue is corrected, which is the PERL program called PrettyLST supplied by the pcgen team.